pytorch/training/docker/2.2/py3/cu121/Dockerfile.gpu

```diff
@@ -478,6 +481,9 @@ RUN /opt/conda/bin/mamba install -y -c conda-forge \
   seaborn \
  && /opt/conda/bin/mamba clean -afy
 
+RUN SMDATAPARALLEL_PT=1 pip install --no-cache-dir ${SMD_DATA_PARALLEL_URL}
```

Comment:
Can you move this installation line to somewhere around line 376 so that any dependencies of smddp binary take precedent over other packages.

Reply:
Just install it right after SM Profiler